.travis.yml

```diff
@@ -13,5 +13,6 @@ install:
 script:
   - TARGET=prod .travis/smoke-test.sh
   - TARGET=dev .travis/smoke-test.sh
+  - .travis/analyse.sh
```

Comment:
are these in an explicit order? If so static checks like `lint` should go before dynamic ones like tests and this?

Reply:
Not really. I'll move this to the end (reordering can happen elsewhere). But they do all run.